mixnet/node.py

```diff
@@ -17,71 +15,50 @@
     UnknownHeaderTypeError,
 )
 
-from mixnet.bls import BlsPrivateKey, BlsPublicKey
 from mixnet.poisson import poisson_interval_sec
-
-NodeId: TypeAlias = BlsPublicKey
-# 32-byte that represents an IP address and a port of a mix node.
-NodeAddress: TypeAlias = bytes
+from mixnet.topology import MixnetTopology, NodeAddress
 
 PacketQueue: TypeAlias = "asyncio.Queue[Tuple[NodeAddress, SphinxPacket]]"
 PacketPayloadQueue: TypeAlias = (
     "asyncio.Queue[Tuple[NodeAddress, SphinxPacket | Payload]]"
 )
 
 
-@dataclass
 class MixNode:
-    identity_private_key: BlsPrivateKey
-    encryption_private_key: X25519PrivateKey
-    addr: NodeAddress
```

Comment:
Moved to `topology.py` with a new name `MixNodeInfo`.

The previous name `MixNode` is now used by `node.py`, which defines the behavior of mix node.